client/src/test/resources/templates/item-content-with-blocks.json

```diff
@@ -145,7 +145,8 @@
           "bodyHtml": "<p>key event text</p>",
           "attributes": {
             "keyEvent": "true",
-            "title": "key event title"
+            "title": "key event title",
+            "pinned": true
```

Comment:
This will actually be a JSON string in the CAPI response, so you will need to add a line to `JsonParser#fixFields` to un-stringify it. Not an ideal solution, but we haven't thought of a better one yet.